tests/Queue/RedisQueueIntegrationTest.php

```diff
@@ -58,7 +58,8 @@ public function setUp()
         }
 
         $this->queue = new RedisQueue($this->redis);
-        $this->queue->setContainer(m::mock(Container::class));
+        $container   = m::mock(Container::class);
```

Comment:
Sorry it's my IDE, usual style for work.
Going to correct it and the last tests that are breaking.